mongo/src/main/java/com/redhat/lightblue/mongo/crud/Translator.java

```
@@ -1258,11 +1260,12 @@ private BasicDBObject objectToBson(JsonNodeCursor cursor, EntityMetadata md) {
             Path path = cursor.getCurrentPath();
             JsonNode node = cursor.getCurrentNode();
             LOGGER.debug("field: {}", path);
-            FieldTreeNode fieldMdNode = md.resolve(path);
-            if (fieldMdNode == null) {
-                throw Error.get(ERR_INVALID_FIELD, path.toString());
+            FieldTreeNode fieldMdNode = null;
+            try {
+                fieldMdNode = md.resolve(path);
+            } catch (Exception e) {
+                return null;
```

This hides the exception. I think removing the try-catch would be the best,